packages/swamp/example/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "swamp-example",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "scripts": {
+    "build": "tsc",
+    "HACK-comment": "TZ=UTC is necessary because SQLite will not respect system timezone",
```

Comment:
It seems // is "idiomatic" https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install

Reply:
Clever